src/plugins/clips-executive/clips/resource-locks.clp

```diff
@@ -150,3 +174,10 @@
               (delete-member$ ?acq (mutex-to-resource ?res))))
   (modify ?m (request NONE) (response NONE))
```

Comment:
Retract resource-locks-resource-requested-for for the unlocked resource at this place, rather than waiting for the goal to switch to another state than COMMITTED